cloudify_tf/terraform/opa.py

```diff
@@ -0,0 +1,246 @@
+import json
+import shutil
+from os import path
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+
+from .tools_base import TFTool, TFToolException
+
+
+class OPA(TFTool):
```

Comment:
```suggestion
class Opa(TFTool):
```

Admittedly, it should probably also be `TfTool`, but that means we need to change the library.